aws/vars.tf

```diff
@@ -145,12 +168,19 @@ variable "distro_ssh_user" {
   default = {
     "debian-stretch" = "admin"
     "debian-buster"  = "admin"
+    "debian-10"      = "admin"
+    "debian-11"      = "admin"
     "fedora-31"      = "fedora"
     "fedora-32"      = "fedora"
+    #"Fedora-Cloud-Base-36" = "fedora"
+    #"Fedora-Cloud-Base-37" = "fedora"
     "ubuntu-bionic"  = "ubuntu"
```

Comment:
Do ssh user names change within a distro? Seems like we could simplify this further and not have to track things by distro major version.

Reply:
I don't even know if we can do fuzzy matching for a map in Terraform. Happy to take a suggestion though!